README.md

```diff
@@ -1,6 +1,23 @@
 # Knowledge Graph
 
-[GraphML](http://graphml.graphdrawing.org/) is XML based format for describing graphs. This schema is understood by chat-gpt. So if we were to give chat-gpt data input compliant to GraphML, it can answer questions around it. GraphML XML-based syntax supports the entire range of possible graph structure constellations including directed, undirected, mixed graphs, hypergraphs, and application-specific attributes
+## Motivation
+ACM is a complex product that is split into many components.  Many components are optional and need to be installed separately.  While we have a lot of good documentation, there are plenty of areas which we do not document well.
+1. product documentation does not include certain kinds of details because they are out of scope
+1. documentation/training materials by nature are siloed because each team produces there own
+1. so we usually do not have good grasp of flows that span teams
```

Comment:
```suggestion
1. Products interactions that span the different features of ACM are not captured well today -- or something like that...
```

Reply:
done. `Products interactions that span the different features of ACM may not be captured well today.`